plugins/inputs/chrony/chrony.go

```diff
@@ -30,26 +40,60 @@ func (*Chrony) SampleConfig() string {
 	return `
   ## If true, chronyc tries to perform a DNS lookup for the time server.
   # dns_lookup = false
+
+  ## Run chronyc binary with sudo.
+  ## Required if chronyd cmdport is disabled, or when running the "serverstats" command.
+  ## Sudo must be configured to allow the telegraf user to run chronyc without a password.
+  ## That configuration may look something like:
+  ## telegraf ALL=(ALL:ALL) NOPASSWD:/usr/bin/chronyc
+  # use_sudo = false
+
+  ## Location of the chronyc binary, if not in PATH
+  # binary = "chronyc"
+
+  ## "tracking" is the default. Some commands with useful fields are "serverstats", "smoothing", "rtcdata", and "ntpdata".
+  ## To run "ntpdata" command for a specific source, append the Name or IP of the source after a space, eg. "ntpdata 10.1.2.3"
+  ## To run "ntpdata" command multiple times for different sources, use multiple instances of this input plugin to allow for adding different tags.
+  # commands = ["tracking"]
```

Comment:
We should create a list of commands we want to support and only allow these commands.  We could use the functions in internal/choice/choice.go to help with this.